### DISCUSSION TITLE
Official discord polls

### DISCUSSION THREAD

**Discussion by DARL0K** [discord-php/DiscordPHP#1219] [Ideas]
Hi Guys. Are you going to add support for Polls in discord?
https://discord.com/developers/docs/resources/poll

**Comment by valzargaming** [comment]
Those of us who generally do PRs are predisposed right now, but it is definitely planned.

**Reply by DARL0K** [reply]
Yeah, I saw your PR. Good to know that you found time at least to start. Thank you for your work!
